### PR TITLE
feat: move chart to the registry

### PR DESCRIPTION
**Pull request by Nox-404**

### CODE REVIEW

File: deploy/scaleway-webhook/templates/deployment.yaml [hunk 1]
```diff
@@ -8,7 +8,9 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  replicas: {{ .Values.replicaCount }}
```
Comment (raskyld) [inline deploy/scaleway-webhook/templates/deployment.yaml:11]: Any reason to remove this param?

Comment (Nox-404) [reply deploy/scaleway-webhook/templates/deployment.yaml:11]: replicaCount wasn't define and I'm not sure we can use multiple replica here

**Review by raskyld — DISMISSED** [review]
LGTM

File: deploy/scaleway-webhook/templates/deployment.yaml (hunk 1 above)
Comment (Nox-404) [reply deploy/scaleway-webhook/templates/deployment.yaml:11]: actually there's no issue with multiple replicas, I'll edit

**Comment by Nox-404** [comment]
closes #11

**Pull request merged**